Tasks/SonarQubePostTest/PRCA/PostComments-Server.ps1

```diff
@@ -46,7 +46,14 @@ function InternalInit
     Assert ( $script:codeReviewClient -ne $null ) "Internal error: could not retrieve the CodeReviewHttpClient object"
                  
     Write-Verbose "Fetching data from build variables"
-    $repositoryId = GetTaskContextVariable "build.repository.id"    
+    $repositoryId = GetTaskContextVariable "build.repository.id"
+
+    # There is a bug in core agent 2.103.0 and lower   
+    if ($repositoryId -eq [Guid]::Empty)
+    {
+        throw "Cannot determine the repository id. Please ensure the agent version is at least 2.103.1"
```

Comment:
Might be better not to use a specific agent version and instead suggest to update. You can also provide the current agent version if it's not easily discoverable and you can actually get it and display it.